README.md

```diff
@@ -1,6 +1,16 @@
 # AEA Ocean-End-To-End
 
-### Dependencies
+## Intro
+
+This agent is responsible for doing a full demo of the main Ocean functionalities.
+The heavy lifting for this use-case is done by the `eightballer/ocean:0.1.0` skill, protocol and connection. 
+
+Once the agent is up & running, it creates the pool & deploys algorithm that will be
```

Comment:
> it creates the pool 

As in the pools we used to use for dynamic pricing? Or a different type of pool?

Reply:
I will figure it out from the code if they use our AMM pools. Thank you for review.

Reply:
From this code, apparently they do not use V4. So there are V3 pools

Reply:
Oh right, that's an interesting choice 🤔